app/services/foreman_discovery/ui_notifications/new_host.rb

```diff
@@ -19,6 +19,7 @@ def add_notification
         Notification.create!(
           initiator: initiator,
           audience: ::Notification::AUDIENCE_SUBJECT,
+          message: _("Host #{subject.name} has been dicovered"),
```

Comment:
Do not use interpolation in i18n messages, please rewrite as noted in: https://projects.theforeman.org/projects/foreman/wiki/Translating